Brain/src/service/train_service.py

```diff
@@ -0,0 +1,53 @@
+"""service to manage trains"""
+from typing import List, Any
+
+from Brain.src.rising_plugin.csv_embed import get_embed
+from Brain.src.service.document_service import DocumentService
+from Brain.src.rising_plugin.pinecone_engine import (
+    get_pinecone_index_namespace,
+    update_pinecone,
+    init_pinecone,
+    delete_pinecone,
+    add_pinecone,
+    delete_all_pinecone,
+)
+
+
+class TrainService:
+    """train (getting embedding) and update pinecone with embeddings by train_id
+    train datatype:
+    key: id
+    values: {id},{data}, {status}"""
+
+    def trainAllDocuments(self) -> None:
```

Comment:
We should validate check in pinecone.  We should prevent the increase of pinecone store with duplicated data. 
There are 2 options to do it: 

1. it is okay to remove all the old trained-data from pinecone and then try to train & insert in it. 
2. it is optional to check if it is existing or not. Then you can update or add.